product_seeds/models/product_product.py

```diff
@@ -45,6 +45,14 @@ def _get_g(self):
         "field.",
     )
     seed_weight = fields.Float(string="Seed Weight", help="Seed Weight")
+    max_seed_weight = fields.Float(string="Maximum Seed Weight", help="Maximum not to be exceeded during bagging.")
+    packing_method_id = fields.Many2one(
+        comodel_name="product.packing.method",
+        string="Packing Method"
+    )
+    theoretical_packing_time = fields.Datetime(string="Theoretical Packing Time")
+    theoretical_adjustment_time = fields.Datetime(string="Theoretical Adjustment Time")
+    legal_mention = fields.Text(string="Legal Mention")
```

Comment:
Will it be different for each product ? Or should it rather be on the company or product category ?